client/src/components/DiagnosticContainer.js

```diff
@@ -39,6 +43,7 @@ const dateOptions = {
 };
 
 function DiagnosticContainer(props) {
+  const [healthLogged] = useState(false);
```

Comment:
This should come from redux, shouldn't it?

Reply:
Yes, it will once we know what redux would look like. For now, it's just for condition underneath

Reply:
We should still get state from Redux whenever possible as it should be the single source of truth for application state, even if it's just placeholder state for now